apps/jobs/tasks.py

```diff
@@ -118,8 +118,7 @@ def _send_transaction(self, signed_tx, eth_action):
             self.async_job.save()
 
 
-@shared_task(bind=True, autoretry_for=(RPCError,),
-             retry_backoff=3, retry_backoff_max=60, max_retries=10)
+@shared_task(bind=True, autoretry_for=(RPCError,), retry_backoff=True, max_retries=None)
```

Comment:
Let's change `retry_backoff_max` to 3600 (60 minutes), I think that is a reasonable max (default is 10min).

Reply:
SGTM, Done.